peerconnection.go

```diff
@@ -526,6 +551,7 @@ func (pc *PeerConnection) createICETransport() *ICETransport {
 			return
 		}
 		pc.onICEConnectionStateChange(cs)
+		pc.updateConnectionState()
```

Comment:
This is called from https://github.com/pion/webrtc/blob/81d03573bcaa2b824c3521d73f16c6c132b6b689/icetransport.go#L188-L195
from https://github.com/pion/webrtc/blob/81d03573bcaa2b824c3521d73f16c6c132b6b689/icetransport.go#L84-L91
from `ice.Agent.updateConnectionState`
https://github.com/pion/ice/blob/bd60240fdc34f14ed511109467a2c930c0b6467e/agent.go#L508-L513

It's called in goroutine and can be parallel.
In very rare case, `pc.iceConnectionState` can be updated twice and then `pc.updateConnectionState()` called twice.
In this case, the first state change may be skipped.

Passing `cs` as an argument of `updateConnectionState()` would be more safe?
(I'm not sure it can be happens actually.)

Reply:
Done! https://github.com/pion/webrtc/pull/830/commits/6ffbea04a93a9c3231fd068a64511936268f2f82

Reply:
I think that makes sense, also just nice to make the function a little simpler.